lib/internal/Magento/Framework/GraphQl/Config.php

```diff
@@ -81,18 +81,24 @@ public function getConfigElement(string $configElementName) : ConfigElementInter
     }
 
     /**
-     * Return all type names declared in a GraphQL schema's configuration.
+     * Return all type names declared in a GraphQL schema's configuration and their type.
      *
-     * @return string[]
+     * @return array $types
+     *  name string
```

Comment:
Is this common style for documenting array structure? Usually I see 
`['name' => 'example value', 'type' = 'example value']`